operators/go/build_subm_operator.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -ex
+
+version=${1:-dev}
+push_image=${2:-false}
+op_gen_dir=$(pwd)
+op_out_dir=$op_gen_dir/submariner-operator
+
+function build_subm_operator() {
```

Comment:
You can just drop this and run the commands :)

Reply:
true

Reply:
done